_layouts/default.html

```diff
@@ -79,6 +82,7 @@
 			<li><a href="/" {% if page.url == "/" %}class="active"{% endif %}>Home</a></li>
 			{% include menu.html %}
 			<li><a href="https://www.npmjs.com/search?q=keywords%3Athelounge-theme">Themes</a></li>
+			<li><a href="https://www.npmjs.com/search?q=keywords%3Athelounge-plugin">Plugins</a></li>
```

Comment:
Let's not include a menu link to plugins for now. The plugin system is way too simple to promote it just yet.